src/main/java/com/main/controller/FollowerController.java

```diff
@@ -0,0 +1,64 @@
+package com.main.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.main.dto.FollowerDto;
+import com.main.dto.ResponseDto;
+import com.main.service.FollowerService;
+
+@RestController
+public class FollowerController {
+
+	private FollowerService followerService;
+
+	public FollowerController(FollowerService followerService) {
+		this.followerService = followerService;
+	}
+
+	@GetMapping(value = { "/followers" })
+	public ResponseEntity<ResponseDto<String>> followers() {
+		Object principal = SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+		String username = "";
+		if (principal instanceof UserDetails) {
```

Comment:
Here the else block is never going to be executed.  Also spring boot has better ways to get the current user, you can do:

```java
@GetMapping(value = { "/followers" })
public ResponseEntity<ResponseDto<String>> followers(@AuthenticationPrincipal UserDetails userDetails) {
    ...
}
```